Makefile

```diff
@@ -160,8 +160,9 @@ xenon:
 
 .PHONY: clean
 clean:
-	@echo "Cleaning project temporary files and installed dependencies..."
-	@docker system prune -a --volumes
+	@echo "Doing docker cleanup for this project..."
+	@docker compose -f docker-compose.yml -f docker-compose.integration-tests.yml down --remove-orphans --volumes --rmi all
```

Comment:
I'm not seeing `-r`, `-m`, and `-i` in the docs (maybe it's `-rm` and `-i`?). If available, can we switch to the long-form in this stored command?

Reply:
`--rmi` information is in the `down` help `docker compose down --help`

```sh
--rmi string        Remove images used by services. "local" remove only images that don't have a custom tag ("local"|"all")
```

Reply:
Thanks - I guess I missed it 🙈